sdk/cosmos/azure-spring-data-cosmos-core/src/main/java/com/azure/spring/data/cosmos/core/query/CosmosQuery.java

```diff
@@ -21,6 +22,8 @@
 
     private Pageable pageable = Pageable.unpaged();
 
+    private PartTree tree;
```

Comment:
We shouldn't bring the PartTree here since the PartTree is a kind of implementation, however, the CosmsQuery is an Abstraction class for CosmosQuery.